chrome/content/zotero/zoteroPane.js

```diff
@@ -4091,13 +4091,38 @@ var ZoteroPane = new function()
 
 	this.buildCopyCollectionMenu = function (event) {
 		if (event.target !== event.currentTarget) return;
-		let popup = event.target;
+		let popup = document.getElementById("zotero-copy-collection-popup");
 		popup.replaceChildren();
 		let selected = this.getSelectedCollection();
 
 		// Fetch all libraries
 		let topLevelEntries = Zotero.Libraries.getAll().filter(lib => !(lib instanceof Zotero.Feed));
 
+		// Check which libraries have collections linked to the selected collection
+		// and disable their menuitems. Same logic as in CollectionTree.canDropCheckAsync.
+		let linkedCollectionsExist = {};
+		(async () => {
+			for (let library of Zotero.Libraries.getAll()) {
```

Comment:
Start with `topLevelEntries` and remove the `instanceof Zotero.Feed` check below?

Reply:
Oh yeah, for sure! I had a glitch with the `async` execution, so I isolated `topLevelEntries` for troubleshooting. That had nothing to do with `topLevelEntries`, so I changed it back to `for (let library of topLevelEntries)`.